wordle-web/src/components/PlayerNameCard.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <v-card
+    class="mt-5 me-5"
+    position="absolute"
+    location="top end"
+    elevation="0"
+    variant="outlined"
+    @click="dialog = !dialog"
+  >
+    <h2 class="pt-2 pe-5 ps-5 pb-2">Player name: {{ props.playerName }}</h2>
+  </v-card>
+  <div class="text-center">
+    <v-dialog v-model="dialog" activator="parent" width="auto">
```

Comment:
You are going to want to add the entered name to local storage, so the player does not need to enter their name each time to access the page.